### DISCUSSION TITLE
Feature?  The examples and effects desperately need some descriptions for them.

### DISCUSSION THREAD

**Discussion by JonnyPhenomenon** [PlummersSoftwareLLC/NightDriverStrip#507] [Ideas]
I know this isn't a bug, so would this be a feature? I don't know.  

I am trying to use the nightdriver for a project where we will use led strips to show the network traffic flowing in and out of our core switch (using snmp and packet counters)  
The issue I am having is that the source code doesn't actually explain what each of the effects or patterns actually do.  
some of the effect.h files say absolutely nothing, while the occasional ones might say something vague like "Draws flying meteors".  - this doesn't tell me how to use the effect, or that the "meteors" are going to bounce back and forth on the strip, etc etc, and I can't tell by looking at the source code because I am a noob, apparently.

sorry.

**Comment by rbergen** [comment]
I'm converting this to an Idea discussion. The presence of the mentioned documentation could possibly facilitate the use of the project by a specific subset of (potential) users, but its absence does not constitute a malfunction or feature that's obviously missing.

**Reply by JonnyPhenomenon** [reply]
Thank you rBergen!
